src/Tizen.Multimedia.MediaPlayer/Player/AudioPitch.cs

```diff
@@ -0,0 +1,125 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Diagnostics;
+using static Interop;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// Provides the ability to control a audio pitch for <see cref="Multimedia.Player"/>.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class AudioPitch
```

Comment:
I recommend to use struct
and C# layer is working only data container

Described in code
``` c#
public struct AudioPitch {
    public bool IsEnabled {get;set;}
    public float Pitch {get;set;}
}

public class Player {
      public AudioPitch AudioPitch {get;set;}
}

player.AudioPitch = new AudioPitch {
    IsEnabled = true,
    Pitch = 3.3
};
```

Reply:
Is there any difference between AudioPitch and SeriesInformation?
SeriesInformation was just data container for two value together.
(SeriesInformation : https://github.com/Samsung/TizenFX/pull/769 )

Reply:
SeriesInformation is immutable object, but current AudioPitch is not

Reply:
I'm confused a little bit.
`Immutable object` is the condition to being `Struct`, Isn't it?

X AVOID defining a struct **unless** the type has all of the following characteristics:
```
- It logically represents a single value, similar to primitive types (int, double, etc.).
- It has an instance size under 16 bytes.
- **It is immutable.**
- It will not have to be boxed frequently.
```

Reply:
So, My means, AudioPitch should be immutable

Reply:
In my opinion, AudioPitch is considered an immutable object.
because it is a entity class(data class) 
It is just set of data to apply Player object
and it has not operator to change internal state. 
So it should be a immutable object.

Reply:
AudioPitch object can be assignable to all others player object
because it is just a data that contained a pitch value

Reply:
If you do not think so,
This is no different than simply attaching it directly to the player.

like below 
``` c#
class Player { ...
 public float AudioPitch {get;set;}
 public bool AudioPitchEnabled {get;set;}
...
}
```

Reply:
`SeriesInformation` is same condition you said the point of view( data container and immutable). 
I just wanna check it again whether `SeriesInformation` should be `Class` or `Struct`.
How about it? I wanna know exactly the criteria for future work.

Reply:
Acutally, SeriesInformation also better to use struct
(SeriesInformation is a data container and immutable)

Reply:
Actually, `SeriesInformation` was `struct` originally but you said it should be `class` because usually use `class`. (https://github.com/Samsung/TizenFX/pull/769#discussion_r270718868)

So now, I see the criteria for choosing between `class` and `struct` clearly. 
Thank you.

Reply:
IsEnabled must be set before player is prepared. each property has a different precondition. nevertheless, should i change class to struct?

Reply:
How about it?
``` c#
class Player { ...
 public float AudioPitch {get;set;}
 public bool AudioPitchEnabled {get;set;}
...
}
```
If it has a different precondition, do not put it in the same value object.

Reply:
I'm worried about current implementation in below usage.
``` c#
AudioPitch pitch = player.AudioPitch;
.....
pitch.Value = 1.0;     // player internal state was changed
pitch.IsEnabled = false; // player internal state was changed
```
I think that control of player should be made directly to player object.
However, current implementation can change the properties of player objects through in different objects like the above example.
AudioPitch object did not represent a player object.
It make hard to debug when complexed issue was occurred

Reply:
hmm. ok i'll check if there is no expandability for AudioPotch more. if so, i'll put these properties into player class.